pkg/render/kube-controllers.go

```diff
@@ -283,6 +289,14 @@ func (c *kubeControllersComponent) controllersRole() *rbacv1.ClusterRole {
 				Verbs:     []string{"create"},
 			})
 		}
+
+		if c.managementClusterConnection != nil {
+			role.Rules = append(role.Rules, rbacv1.PolicyRule{
```

Comment:
How does this work since the actual create/update comes from the management cluster?
Does guardian impersonate kube-controller in the managed cluster, so operator in the managed cluster needs to give kube-controller permissions to create/update the licensekeys even though the kube-controller in the managed cluster won't be doing anything with licenses.

Reply:
Yes, all API requests that pass through the tunnel use impersonation.

Reply:
That is right. Guardian impersonates ckc and for this the role should be present.